client/src/app/gui.rs

```diff
@@ -63,6 +63,7 @@ pub struct Gui {
     rtp_packets_table: RtpPacketsTable,
     rtp_streams_table: RtpStreamsTable,
     rtp_streams_plot: RtpStreamsPlot,
+    update_counter: u32,
```

Comment:
Is this `update_counter` used for anything or you just forgot to remove it with the "refresh every x frames" function? It's used in some other places as well.

Reply:
thanks 
<img width="132" alt="image" src="https://github.com/LVala/rtpeeker/assets/64365037/c443f028-4ca9-431d-8df4-20c27974c0ae">